sound/soc/intel/boards/sof_sdw.c

```diff
@@ -232,6 +257,18 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 		.ops = &sof_sdw_rt1308_i2s_ops,
 		.init = sof_sdw_rt1308_init,
 	},
+	{
+		.part_id = 0x1316,
+		.direction = {true, true},
```

Comment:
so 1316 supports capture ?

Reply:
> so 1316 supports capture ?

Yes, it has speaker protection feature.

Reply:
so, new problem is coming.  715 is fixed to SDW_DMIC_DAI_ID ( = 4), but now for case 711-1316-1316-715, there will be 2 + 2 + 2 + 1 BE DAI, much more than 711-1308-1308-715 (2 + 1 + 1 + 1).   And now do we need to fix 715-SDCA ? And fix to which one ? If It depends on topology, so we need new topology since BE DAI changes.

````c
	/*
	 * DAI ID is fixed at SDW_DMIC_DAI_ID for 715-SDCA to
	 * keep sdw DMIC and HDMI setting static in UCM
	 */
	if (sof_sdw_quirk & SOF_RT715_DAI_ID_FIX)
		dai_links->id = SDW_DMIC_DAI_ID;
````

Reply:
> so, new problem is coming. 715 is fixed to SDW_DMIC_DAI_ID ( = 4), but now for case 711-1316-1316-715, there will be 2 + 2 + 2 + 1 BE DAI, much more than 711-1308-1308-715 (2 + 1 + 1 + 1). And now do we need to fix 715-SDCA ? And fix to which one ? If It depends on topology, so we need new topology since BE DAI changes.

Two 1316s are in a group. So there will be 2+2+1 BE DAI.

Reply:
If so, we don't need to fix DAI ID, it is already right. And we still can't reuse the topology for 711-1308-1308-715

Reply:
@RanderWang did you mean we *can* reuse the same topology?